File: Iniciante/1038.py
C, Q = input().split()

C = int(C)
Q = int(Q)

if(C==1):
    V = Q * 4.00
    print(f'Total: R$ {V:.2f}')
elif(C==2):
    V = Q * 4.50
    print(f'Total: R$ {V:.2f}')
elif(C==3):
    V = Q * 5.00
    print(f'Total: R$ {V:.2f}')
elif(C==4):
    V = Q * 2.00
    print(f'Total: R$ {V:.2f}')
elif(C==5):
    V = Q * 1.50
    print(f'Total: R$ {V:.2f}')

File: Iniciante/1038.php
<?php

$L1 = explode(" ", readline());
$c = $L1[0];
$q = $L1[1];

if($c==1){
    $v = number_format($q * 4.00, 2, '.', '');
    echo "Total: R$ $v\n";
}elseif($c==2){
    $v = number_format($q * 4.50, 2, '.', '');
    echo "Total: R$ $v\n";
}elseif($c==3){
    $v = number_format($q * 5.00, 2, '.', '');
    echo "Total: R$ $v\n";
}elseif($c==4){
    $v = number_format($q * 2.00, 2, '.', '');
    echo "Total: R$ $v\n";
}elseif($c==5){
    $v = number_format($q * 1.50, 2, '.', '');
    echo "Total: R$ $v\n";
}

?>